companies/zucchetti.json

```diff
@@ -0,0 +1,26 @@
+{
+    "slug": "zucchetti",
+    "relevant-countries": [
+        "it",
+        "us",
+        "spa",
+        "br",
+        "fr"
+    ],
+    "categories": [
+        "it",
```

Comment:
Only the [categories specified in the schema](https://github.com/datenanfragen/data/blob/master/schema.json#L30) are allowed.